src/components/AuthenticationModal.tsx

```diff
@@ -113,12 +117,16 @@ const ConfirmWithAuthModal = ({ onConfirm, onClose, usePin = false }: ConfirmWit
           <CenteredInstructions instructions={shownInstructions} />
           <PinCodeInput pinLength={pinLength} onPinEntered={decryptMnemonic} />
         </ModalContent>
+      ) : (
+        <Message>
+          <AppText>Loading wallet...</AppText>
+        </Message>
```

Comment:
To make sure there is never just a white screen. Should only be displayed while the fingerprint sensor is visible.